### DISCUSSION TITLE
Conceptos de Algoritmos Genéticos y Redes Neuronales

### DISCUSSION THREAD

**Discussion by RodrigoMijangos** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#521] [Ideas]
# Hola Viajeros. 
Desgraciadamente he estado bastante ausente en el curso debido a que he realizado prácticas de IA para la escuela y justo ahora estoy codificando una IA con mi equipo mediante Tensorflow que permita jugar contigo y ganarte o empatarte dependiendo de lo que pongas. Así que para compensar mi tiempo ausente he decidido intentar darles una pequeña explicación sobre el proceso creativo que tuve al momento de diseñar Redes Neuronales con Python y Algoritmos genéticos. Espero pronto ponerme al corriente y les dejo 2 repositorios mios en donde está todo lo que he logrado hacer de estas redes.

Generación de Datasets para Tensorflow: [https://github.com/RodrigoMijangos/datasets_tablerso_gato](url)
Neurona: [https://github.com/RodrigoMijangos/NN_27_FEB_2022](lurl)

Por ahora solo tengo la generación de los datasets, tomenlo como una muestra de que se puede modelar cualquier problema en un conjunto de datos que pueda ser interpretado por una máquina.

Por ahora es todo. Espero verlos pronto viajeros.
**Hasta el infinito y más allá**

**Comment by AngelCruzO** [comment]
El tema es bastante interesante, la IA resuelve muchos problemas de procesamiento que la programación clásica no puede. Yo en mi clase de computación evolutiva, intenté realizar el n-reinas en C. Y la verdad programar IA con python es mi chismosa más sencillo
